### PR TITLE
LP-2477 toggle eye icon classes

### PR DESCRIPTION
**Pull request by asadali145**

### CODE REVIEW

File: openedx/adg/lms/static/components/student_account/PasswordResetConfirmation.jsx [hunk 1]
```diff
@@ -41,6 +44,8 @@ class PasswordResetConfirmation extends React.Component {
       validationMessage: '',
       passwordType1: 'password',
       passwordType2: 'password',
+      eyeIconClasses1: this.hidePasswordClasses,
+      eyeIconClasses2: this.hidePasswordClasses,
```
Comment (zeshanarifios) [inline openedx/adg/lms/static/components/student_account/PasswordResetConfirmation.jsx:48]: please specify hide in keys as well.

File: openedx/adg/lms/static/components/student_account/PasswordResetConfirmation.jsx [hunk 2]
```diff
@@ -52,15 +57,21 @@ class PasswordResetConfirmation extends React.Component {
       return (passwordType === 'password') ? 'text' : 'password'
   }
 
+  getEyeIconClasses(prevClasses) {
+      return (prevClasses.includes('show-password')) ? this.hidePasswordClasses : this.showPasswordClasses
```
Comment (zeshanarifios) [inline openedx/adg/lms/static/components/student_account/PasswordResetConfirmation.jsx:61]: How about defining a variable like `isPasswordVisible = prevClasses.includes('show-password')` and use it in condition to make condition more readable

**Review by zeshanarifios — CHANGES_REQUESTED** [review]

File: openedx/adg/lms/static/components/student_account/PasswordResetConfirmation.jsx (hunk 1 above)
Comment (asadali145) [reply openedx/adg/lms/static/components/student_account/PasswordResetConfirmation.jsx:48]: This can be show password classes as well. Can't change that.

**Review by zeshanarifios — APPROVED** [review]
LGTM 👍

**Review by ahmed-faraz46 — APPROVED** [review]

**Pull request merged**